### ISSUE TITLE
Setup test to verify non-SRD information is not included.

### ISSUE THREAD

**Issue by benjaminapetersen** [5e-bits/5e-database#114]
I believe Travis-CI will run for free against a github repo.  As long as PRs are used rather than direct merge to master, a simple test script could be written to read the JSON files, and validate the items.  
For example, load the `monsters-SRD.json`, loop all the `names` and verify a hard-coded list of `names`.  If anyone adds additional, tests fail.

**Comment by bagelbits** [comment]
That sounds like a pretty good idea.

**Comment by bagelbits** [comment]
@benjaminapetersen Let me know if I can help with this as at all.

**Comment by benjaminapetersen** [comment]
The main issue here is the need to run some code.  Does anyone have strong feelings on language, etc?

**Comment by bagelbits** [comment]
Not really though I think the scripts that currently exist in this repo are
all JS. Though I think they need to be organized and maybe documented.

**Comment by bagelbits** [comment]
@ogregoire or @Alex-frazer Strong feelings on language? I mostly work in Ruby and Node. But for consistency between the languages currently in the DB and API, Node might make the most sense for now?

**Comment by benjaminapetersen** [comment]
Node is probably the most approachable to the most people.

**Comment by ogregoire** [comment]
I'm more of a C/C++/Java/Python guy, but any language is good for me as it will be quite easy to implement.

**Comment by bagelbits** [comment]
I just cut #132  which should lay some of the ground work for some of this. At least the bare minimum of a linter. I can also handle setting up the plumbing for Buildkite so that we can run that. Thought that'll have to wait until ownership gets transferred.

**Comment by bagelbits** [comment]
Yup. #132 is ready to go and it will also add in Travis CI. I'm also happy to split up the PR, as it's doing quite a bit of stuff.

**Comment by benjaminapetersen** [comment]
+1

**Comment by bagelbits** [comment]
@benjaminapetersen Travis  CI is now setup.

**Comment by bagelbits** [comment]
Closing: CI and schema/table tests now run on every PR. A hard-coded SRD name allowlist would be costly to maintain across 2014, 2024, and translations; PR review covers non-SRD content.
